src/python/WMCore/Services/Dashboard/Logger.py

```diff
@@ -0,0 +1,78 @@
+"""
+ * ApMon - Application Monitoring Tool
+ * Version: 2.2.20
+ *
+ * Copyright (C) 2006 California Institute of Technology
+ *
+ * Permission is hereby granted, free of charge, to use, copy and modify
+ * this software and its documentation (the "Software") for any
+ * purpose, provided that existing copyright notices are retained in
+ * all copies and that this notice is included verbatim in any distributions
+ * or substantial portions of the Software.
+ * This software is a part of the MonALISA framework (http://monalisa.cacr.caltech.edu).
+ * Users of the Software are asked to feed back problems, benefits,
+ * and/or suggestions about the software to the MonALISA Development Team
+ * (developers@monalisa.cern.ch). Support for this software - fixing of bugs,
+ * incorporation of new features - is done on a best effort basis. All bug
+ * fixes and enhancements will be made available under the same terms and
+ * conditions as the original software,
+
+ * IN NO EVENT SHALL THE AUTHORS OR DISTRIBUTORS BE LIABLE TO ANY PARTY FOR
+ * DIRECT, INDIRECT, SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES ARISING OUT
+ * OF THE USE OF THIS SOFTWARE, ITS DOCUMENTATION, OR ANY DERIVATIVES THEREOF,
+ * EVEN IF THE AUTHORS HAVE BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+ * THE AUTHORS AND DISTRIBUTORS SPECIFICALLY DISCLAIM ANY WARRANTIES,
+ * INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE, AND NON-INFRINGEMENT. THIS SOFTWARE IS
+ * PROVIDED ON AN "AS IS" BASIS, AND THE AUTHORS AND DISTRIBUTORS HAVE NO
+ * OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES, ENHANCEMENTS, OR
+ * MODIFICATIONS.
+"""
+from __future__ import print_function
+from __future__ import division
+from builtins import object
```

Comment:
To be more clear, the problem occurs since the code is running in worker node. If the code is run agent there is no problem, since agent rpm has dependency on future package. But it doesn't seem that package is ported to worker node.  Anyway, I will create a new issue for that.